community/contribute/release.md

```diff
@@ -78,3 +78,14 @@ the release steps described above, it will perform following actions;
 - Updates `./versions.json` file,
 - Creates new sidebar in `./versioned_sidebars`,
 - Creates an automatic PR against `main` branch
+
+### Archiving the older versions
+
+Currently, archiving older versions is manual. To archive an older version, follow these steps:
+
+- Remove the version from `versions.json`
+- Remove all files under `versioned_docs/version-{version}`
+- Remove sidebar file under `versioned_sidebars/version-{version}-sidebars.json`
```

Comment:
These 2 steps are not clear for me.
In this branch I can see old versions 0.13, 0.14 etc

Reply:
Yes, because it's a separate PR to remove thse files: https://github.com/kubeshop/botkube-docs/pull/271